src/main/java/io/personium/core/model/DavCommon.java

```diff
@@ -41,25 +41,29 @@ private DavCommon() {
     /** Overwrite Value when header overwrite is not allowed.*/
     public static final String OVERWRITE_FALSE = "F";
 
+    public static Pattern PATTERN_CTRL_CHARS = Pattern.compile("\\p{Cntrl}");
```

Comment:
Sorry, `\p{Cntrl}` should be used with `(?U)` flag.
I think `\p{Cc}` should be used. 

```suggestion
    public static Pattern PATTERN_CTRL_CHARS = Pattern.compile("\\p{Cc}");
```